package.json

```diff
@@ -20,12 +20,12 @@
   },
   "dependencies": {
     "keyb": "^1.1.0",
-    "pixi.js": "^4.5.5",
     "statgrab": "^1.2.0",
     "yaafloop": "^1.1.0"
   },
   "devDependencies": {
     "babel-preset-es2015": "^6.24.1",
-    "codeglue": "^1.3.0"
+    "codeglue": "^1.3.0",
+    "pixi.js": "^4.5.6"
```

Comment:
Do we need `pixi.js` as a dev-dependency even? We're using an external cloud host for it now, right?

Reply:
My thinking was that it would sort of satisfy the one case against the CDN in local dev... working offline. So with pixijs in the dev dependencies if you are online and install, if you are later working madly on something and the internet goes out (or the CDN), you could change the path to the local version and carry on.

Easy to remove though if it doesn't seem worth it.

Reply:
Or copy/paste into the dev server root and update the path I guess.